index.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		commonjs: true,
+		es6: true,
+		node: true,
+	},
+	globals: {
+		t: true,
```

Comment:
shall we still keep this?

Reply:
I don't think so. We should encourage people to use `nextcloud-l10n` and add `t` and `n` to Vue's prototype (or just import in every module needed)

Reply:
I think using an import is enough! Let's do this :)

Reply:
@ChristophWurst same for OCP/OCA/OC? I guess we cannot know what is fully under OCA/OCP, but we can make sure people do not use OC ?

Reply:
Not everything has been ported to our npm packages yet, so for now i would just keep it. :)

Reply:
> I don't think so. We should encourage people to use `nextcloud-l10n` and add `t` and `n` to Vue's prototype (or just import in every module needed)

Vue recommends to scope the values to bind with `Vue.prototype` using `$`: https://vuejs.org/v2/cookbook/adding-instance-properties.html#The-Importance-of-Scoping-Instance-Properties

So it should be `Vue.prototype.$t = t`

Reply:
I find the import more appropriate. But can you use t in templates after importing? Or do you need some kind of weird mixin and/or computed?
Of so we should indeed go for $t instead!

Reply:
You will still have to use computed properties as far as i know.

Reply:
Ok, so let's move to $t directly! cleaner!

Reply:
We have globals already defined in https://github.com/nextcloud/eslint-plugin-nextcloud/blob/master/lib/environments/nextcloud.js - this all is not needed, here 😕